### PR TITLE
Release edits for cromshell2

### PR DESCRIPTION
**Pull request by bshifaw**
- initiating cromshell2 would be `cromshell` instead of `cromshell-beta`
- Added shorthand for setting the cromwell url to be `--cu`
- minor grammar changes to ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,36 +7,28 @@
       ""     ""    ""                                                          ""    ""     ""
 ```
 
-# cromshell
- A CLI for submitting workflows to a cromwell server and monitoring / querying their results.
+# Cromshell
+[![GitHub version](https://badge.fury.io/gh/broadinstitute%2Fcromshell.svg)](https://badge.fury.io/gh/broadinstitute%2Fcromshell)
```
Comment (bshifaw) [inline README.md:11]: Replaced the "Current version:..." wording with a version badge. There are other badges that can be added later like coverage

[codecov](https://codecov.io/gh/broadinstitute/cromshell)

File: README.md [hunk 2]
```diff
@@ -7,36 +7,31 @@
...
+[![Integration Workflow](https://github.com/broadinstitute/cromshell/actions/workflows/integration_tests.yml/badge.svg)](https://github.com/broadinstitute/cromshell/actions/workflows/integration_tests.yml/badge.svg)
+[![Integration Workflow](https://github.com/broadinstitute/cromshell/actions/workflows/unit_tests.yml/badge.svg)](https://github.com/broadinstitute/cromshell/actions/workflows/unit_tests.yml/badge.svg)
```
Comment (lbergelson) [inline README.md:13]: This is called integration workflow twice, I think one is unit test?

File: README.md [hunk 3]
```diff
@@ -90,19 +84,19 @@ functions as Cromshell 1 but has been rebuilt in python with many added benefits
    
    ####  Get email notification on job completion
```
Comment (lbergelson) [inline README.md:85]: Should we pull this from the readme for now?

File: README.md [hunk 4]
```diff
@@ -7,36 +7,31 @@
...
+         cromshell -t 20 metadata
+         cromshell logs -2
 ```
 
 ## Supported Options:
   * `--no_turtle` or `--I_hate_turtles`
     * Hide turtle logo
   * `--cromwell_url [TEXT]`
-    * Specify Cromwell URL used. 
+    * Specifies Cromwell URL used.
```
Comment (bshifaw) [inline README.md:31]: mention some cromwell servers the port specification isn't required

Comment (bshifaw) [reply README.md:31]: done

**Pull request merged**